sql/expression/function/linestring.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020-2021 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package function
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/dolthub/go-mysql-server/sql"
+)
+
+// Linestring is a function that returns a point type containing values Y and Y.
+type Linestring struct {
+	args []sql.Expression
+}
+
+var _ sql.FunctionExpression = (*Linestring)(nil)
+
+// NewLinestring creates a new point expression.
+func NewLinestring(args ...sql.Expression) (sql.Expression, error) {
+	if len(args) < 1 {
```

Comment:
what is the minimum number of points in a linestring?

Reply:
2